src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.jsx

```diff
@@ -94,72 +93,59 @@ const SelectHareDropDown = ({
       ],
   );
 
-  const { status: statusX, data: dataX } = hareResults.at(
-    hareResults.length === 1 ? 0 : -1,
-  );
-
-  useEffect(() => {
-    setHares(dataX?.concept_breakdown);
-  }, [dataX]);
+  const getFinalPopulationSize = (selectedHareItem) => {
+    const hareBreakdownPart1 = hareResults[0].data.concept_breakdown.find((hare) => selectedHareItem.value === hare.concept_value).persons_in_cohort_with_value;
+    if (outcome.variable_type === 'concept') {
+      return hareBreakdownPart1;
+    } else {
+      const hareBreakdownPart2 = hareResults[1].data.concept_breakdown.find((hare) => selectedHareItem.value === hare.concept_value).persons_in_cohort_with_value;
```

Comment:
1.) Are there a more descriptive variable names that could be used here other than hareBreakdownPart1 and hareBreakdownPart1 ? Before we were discussing 
```
Control Size        123
Case Size           456
Total Size          579
```
Is hareBreakdownPart1 the Control Size and part 2 the Case size? If so can the variable names be updated to reflect that information? 

Also, could a line break be added to break up the rather long lines (lines 97 & 101)? I have to scroll to read the function.

Reply:
Yes, I think we should have this in a separate PR (i.e. the breaking down into Case and Control). This PR is just to get the total size, so right now in this PR it is not important to try to figure out which one is Case, which one is Control

Reply:
Long lines are broken up now.